src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
+  const separatorTo = toFormat.pop();
```

Comment:
Similarly to the previous comment, using `pop` on `toFormat` will mutate the original array. It's better to access the last element without removing it.